.circleci/config.yml

```diff
@@ -21,3 +21,6 @@ jobs:
           command: |
             /opt/conda/bin/flake8 --version
             /opt/conda/bin/flake8 obspy
+          name: run file size check
+          command: |
+            git diff --name-only $CIRCLE_COMPARE_URL | xargs -I {} find {} -size +5 | wc -l
```

Comment:
Looks good.. even better would be to check size of all history in the PR, sometimes large files get added and later removed in another commit, still cluttering repo history size.
I tried this, but it seems it's not listing the size of all objects in the respective commit actually.. maybe just as an inspiration..

`for HASH in $(git log --pretty="format:%H" master..); do git cat-file -s $HASH; done`